src/mkdocstrings_handlers/python/rendering.py

```diff
@@ -539,11 +539,20 @@ def do_as_attributes_section(
     Returns:
         An attributes docstring section.
     """
+    def _parse_docstring_description(attribute: Attribute) -> str:
+        if attribute.docstring is None:
+            return ""
+        lines = attribute.docstring.value.lstrip().split("\n", 1)
+        if ":" in lines[0]:
+            _, line = lines[0].split(":", 1)
+            lines = [line, *lines[1:]]
+        return lines[0]
+
     return DocstringSectionAttributes(
         [
             DocstringAttribute(
                 name=attribute.name,
-                description=attribute.docstring.value.split("\n", 1)[0] if attribute.docstring else "",
+                description=_parse_docstring_description(attribute),
```

Comment:
I believe we should split out contents before the first colon `:` *only* if the attribute docstring has the `returns_type_in_property_summary: true` option set in its [`parser_options`](https://mkdocstrings.github.io/griffe/reference/api/docstrings/models/#griffe.Docstring.parser_options) dict. Lets check that inside the function.